crates/matrix-sdk/src/sliding_sync/mod.rs

```diff
@@ -1088,12 +1107,12 @@ impl SlidingSync {
         // Spawn a new future to ensure that the code inside this future cannot be
         // cancelled if this method is cancelled.
         spawn(async move {
-            debug!("Sliding sync response received");
+            debug!("Sliding Sync response handing starts");
 
             // In case the task running this future is detached, we must be
-            // ensure responses are handled one at a time, hence we lock the
+            // ensured responses are handled one at a time, hence we lock the
```

Comment:
Not sure that this sounds better to me. I think removing the "be" would improve it, i.e. "we must ensure" sounds correct to me.

Reply:
Correct :-)!